gsa/src/web/pages/tlscertificates/details.js

```diff
@@ -68,15 +72,31 @@ const TlsCertificateDetails = ({entity, links = true}) => {
               <TableData>{renderYesNo(entity.valid)}</TableData>
             </TableRow>
           )}
+          {isDefined(entity.activationTime) && (
+            <TableRow>
+              <TableData>{_('Activates')}</TableData>
+              <TableData>
+                {dateTimeWithTimeZone(entity.activationTime)}
+              </TableData>
+            </TableRow>
+          )}
+          {isDefined(entity.expirationTime) && (
+            <TableRow>
+              <TableData>{_('Expires')}</TableData>
+              <TableData>
+                {dateTimeWithTimeZone(entity.expirationTime)}
```

Comment:
Same here. It should use the DateTime component instead of the dateTimeWithTimeZone function